csharp/README.md

```diff
@@ -176,12 +161,6 @@ If everything worked out, you should see lots of `ok` messages. In case of a
 failure, the tests abort with `failed`.
 
 On Windows, `allTests.py` executes by default the tests for .NET Framework 4.5.
-In order to execute the tests with .NET Core framework add the `--dotnetcore`
```

Comment:
I think it would be clearer for the default to be the same on all platforms.

And if for some reason that's too hard, document the default on all platforms, e.g.


`allTests.py` executes the tests for the default framework: `net45` on Windows and `net6.0` on other platforms.

Reply:
Ok, I will switch to .NET 6.0 as default for all platforms